From 0ac4182d43e00bdf0e3f31fc38860aa0ee680f76 Mon Sep 17 00:00:00 2001
From: Zhantao Tang <zhantao.tang@windriver.com>
Date: Mon, 1 Mar 2021 16:21:10 +0800
Subject: [PATCH] dts: s32g: modify the hse_reserved memory node to compitable
 with hse firmware

In latest uboot version bsp28, the dts file for s32g274aevb need to
be modified to compitable with the hse firmware to make hse function
works.

And thus the related s32gen1_secboot.sh should be modified too, to
change UBOOT_MAX_SIZE value for u-boot.s32 file.

Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm/dts/fsl-s32g274aevb.dts | 4 ++--
 tools/s32gen1_secboot.sh         | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/dts/fsl-s32g274aevb.dts b/arch/arm/dts/fsl-s32g274aevb.dts
index 958cffd3c2..7246905631 100644
--- a/arch/arm/dts/fsl-s32g274aevb.dts
+++ b/arch/arm/dts/fsl-s32g274aevb.dts
@@ -18,9 +18,9 @@
 		ranges;
 
 		/* reserved space to store hse data */
-		hse_reserved: hse_reserved@0x80000000 {
+		hse_reserved: hse_reserved@0xd0000000 {
 			no-map;
-			reg = <0 0x80000000 0 0x40000>; /* 256K */
+			reg = <0 0xd0000000 0 0x40000>; /* 256K */
 		};
 	};
 #endif
diff --git a/tools/s32gen1_secboot.sh b/tools/s32gen1_secboot.sh
index 1b2c4174bd..b2b5fa8913 100755
--- a/tools/s32gen1_secboot.sh
+++ b/tools/s32gen1_secboot.sh
@@ -41,7 +41,7 @@ while [[ $# -gt 0 ]]; do
 	shift
 done
 
-UBOOT_MAX_SIZE=1048576 
+UBOOT_MAX_SIZE=1500000
 
 # switch to script dir
 SCRIPT_PATH=${0%/*}
-- 
2.17.1

